src/configParser.js

```diff
@@ -21,12 +21,16 @@ ConfigParser.prototype.mergeConfigs = function (config) {
   // apply common profile
   this._mergeConfig('../conf/profile.conf.js', 'common profile');
 
+  // Changing config via confKeys
+  this._setConfKeys();
   // return new fully merged config
   return this.config;
 };
 
 ConfigParser.prototype._mergeConfig = function (configFile, type) {
   this.logger.debug('Loading ' + type + ' config from: ' + configFile);
+  // Need to remove cashed files from previous merges 
+  delete require.cache[require.resolve(configFile)];
```

Comment:
Is that really necessary, we merge the configs only once per execution ?

Reply:
Not in tests though, I wasn't sure if it is done like in tests somewhere else, so just to be sure I dropped cache here. 
It might be enough to drop it in tests though.

Reply:
In tests we work with mocked data directly with api so I do not think this is necessary there also. Can you please remove this ?